### DISCUSSION TITLE
Change Double Click Behavior

### DISCUSSION THREAD

**Discussion by timothyperando** [microsoft/azuredatastudio#25281] [Q&A]
I'm wondering if anyone has figured out how to set up Azure Data Studio to select the whole column name with the brackets when double clicking on the column name? I know SSMS does this by default, but I would like to see if Azure Data Studio can do the same thing.

For example: If my column name is [Sequence Number 1] and I double click on the N in Number it selects only the word Number in Azure Data Studio. In SSMS it selects the whole column name plus the brackets when double clicking.

I've been through the settings tab, but I'm not seeing this option and I'm not able to find anything in online searches.

**Comment by IT-CASADO** [comment]
An option would be fine, but for me it should be ON by default. The current behavior is very annoying.

Another annoying selection behavior of same category is following:

```SQL
SELECT 
       [d]
      ,[Name]
  FROM [Table]
```

1. Position the cursor after [Name]
2. Press SHIFT+CTRL+LEFT ARROW on the keyboard for selecting the column name

In that case the left bracket is not select.

If you press SHIFT+CTRL+LEFT ARROW twice the comma is also selected :(
